README.rst

```diff
@@ -116,6 +119,52 @@ Usage
       latest     Get the latest tag in a changelog.
       tag        Create an annotated git tag based on a changelog entry.
       update     Updates the version and date of the most recent changelog entry.
+      create     Creates a new CHANGELOG.md file with a standard header.
+      entry      Add a quick entry to the current version.
+      next       Create a new unreleased version in changelog file.
+
+next
+----
+
+Create a new unreleased version in changelog file.
+
+::
+
+   Usage: chag next
+
+   Creates a new unreleased version inside of the changelog file.
+
+   Options:
+     --help     Displays this message.
+
+entry
```

Comment:
Your pull request has not been merged after more than a year and that's why I make little to no assumptions whether this is going to happen soon or anytime at all. If I did not move on I would still wait for your changes to be merged into the original repository. Thus I would have to use two versions of chag to use all the features of both forks. This is one of the main reasons why forking and permissive licensing of creative work exist.

Because of that I pulled and merged your work into my fork because to me it looked good, it works and it eases my day to day work. I further improved `chag` and it would now be impossible to rebase again without producing merge conflicts. Given that all your work has been merged into the other fork and all the work in that fork is available as a pull request it is indeed the case that pull request #10 supersedes this pull request. There's no offense or aggressive behavior behind that. It simply states that the work of this pull request is included in another pull request. If you think that forking is "hijacking" (a synonym to "stealing") of your creative work, please re-read the MIT license. To me this discussion ends here and I am going back to work on something productive like improving the usability of chag.